usd/src/usd_parser/USDWorld.cc

```diff
@@ -50,10 +62,65 @@ namespace usd
     }
     _world->worldName = reference->GetDefaultPrim().GetName().GetText();
 
+    std::string linkName;
+
     auto range = pxr::UsdPrimRange::Stage(reference);
     for (auto const &prim : range)
     {
+      // Skip materials, the data is already available in the USDData class
+      if (prim.IsA<pxr::UsdShadeMaterial>() || prim.IsA<pxr::UsdShadeShader>())
+      {
+        continue;
+      }
+
       std::string primName = prim.GetName();
+      std::string primPath = pxr::TfStringify(prim.GetPath());
+      std::string primType = pxr::TfStringify(prim.GetPath());
```

Comment:
It seems strange to me that `primPath` and `primType` are identical, maybe there is a mistake here?